.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build pull request
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/pr-build.yml"
+      - "app/**"
+      - "gradle/**"
+      - "build.graddle.kts"
+      - "gradle.properties"
+      - "settings.gradle.kts"
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Make sure the release step uses its own credentials:
+          # https://github.com/cycjimmy/semantic-release-action#private-packages
+          persist-credentials: false
+          fetch-depth: 0
```

Comment:
What are these for?

Reply:
I do not know but it is on the other workflow.
[pr-build](https://github.com/ReVanced/revanced-manager/blob/main/.github/workflows/pr-build.yml#L16)

I think it might be safe to remove.